Makefile

```diff
@@ -35,10 +35,12 @@ ICONNAME=stax_app_sia.gif
 endif
 
 # The --path argument here restricts which BIP32 paths the app is allowed to derive.
-APP_LOAD_PARAMS = --path "44'/93'" --curve ed25519 $(COMMON_LOAD_PARAMS)
+APP_LOAD_PARAMS = --path "44'/93'" --curve secp256k1 --curve ed25519 $(COMMON_LOAD_PARAMS)
```

Comment:
why do we need secp256k1?

Reply:
It didn't make sense to me but some Ledger tool was warning about it.  Apparently https://github.com/LedgerHQ/ledger-app-database/blob/main/app-load-params-db.json#L1194 is wrong.  I guess we could make a PR there?

Reply:
https://github.com/LedgerHQ/ledger-app-database/pull/27